devel/inspect-screen.lua

```diff
@@ -255,7 +255,14 @@ local function get_map_report(show_empty)
     local report = {}
 
     local pos = cur_mouse_pos
-    if not pos or pos.x < 0 or pos.y < 0 or not pos.z or pos.z < 0 then
+    if not pos or not pos.z then
```

Comment:
Do we not need the `< 0` checks anymore?

Reply:
No, I fixed the bug in Gui::getMousePos() that was allowing that condition to occur in the first place

Reply:
Oh, I think I see, the `gui_getMousePos()` wrapper in LuaApi should be returning nil for invalid coordinates. Guessing the fix is in https://github.com/DFHack/dfhack/pull/2531/files#diff-a3cdb8e239e182d28f7a044fde4b571eac34f81293c8b471d43d512b129c0602R2152

Reply:
that's right